client/components/forms/form-input-validation/index.jsx

```diff
@@ -1,26 +1,33 @@
 /**
  * External dependencies
  */
-var React = require( 'react/addons' ),
-	classNames = require( 'classnames' );
+import React from 'react/addons';
+import classNames from 'classnames';
+
+/**
+ * Interal dependencies
+ */
+import Gridicon from 'components/gridicon';
 
 module.exports = React.createClass( {
 
 	displayName: 'FormInputValidation',
 
-	getDefaultProps: function() {
+	getDefaultProps() {
 		return { isError: false };
 	},
 
-	render: function() {
-		var classes = classNames( {
+	render() {
+		const classes = classNames( {
 			'form-input-validation': true,
 			'is-error': this.props.isError
 		} );
 
+		const icon = this.props.isError ? 'notice-outline' : 'checkmark';
```

Comment:
@rickybanister mentioned:

> It would be nice if we could modify it to allow arbitrary icons to be passed.

I don't think we have to accomplish that in this PR, but something I thought I'd go ahead and point.